lib/engine/game/g_1844/entities.rb

```diff
@@ -60,7 +60,7 @@ module Entities
               {
                 type: 'choose_ability',
                 after_phase: '2',
-                when: %w[track owning_player_track],
+                when: 'any',
```

Comment:
I'd like to still limit this ability as I feel `any` is obnoxious to deal with and we can limit to a short list that does not affect game play.